packages/marqus-desktop/webpack.rules.js

```diff
@@ -24,4 +24,8 @@ module.exports = [
       "sass-loader",
     ],
   },
+  {
+    test: /\.ttf$/,
+    use: [{ loader: "file-loader" }],
+  },
```

Comment:
When using webpack 4 with the monaco-editor-webpack-plugin it's required to use `file-loader`
https://github.com/microsoft/monaco-editor/tree/main/webpack-plugin